frontend/pages/queries/edit/components/SaveQueryModal/SaveQueryModal.tsx

```diff
@@ -315,7 +238,17 @@ const SaveQueryModal = ({
               label="Logging"
               wrapperClassName={`${baseClass}__form-field ${baseClass}__form-field--logging`}
             />
-            {!isLoadingAppConfig && renderDiscardDataOption()}
+            {!isLoadingAppConfig && (
+              <DiscardDataOption
+                {...{
```

Comment:
Is this ... necessary? Aren't these just props that can be set directly?

Reply:
^